src/Adapters/WebForms/Controls/RouteLinkCapability.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using DotVVM.Framework.Binding;
+
+namespace DotVVM.Adapters.WebForms.Controls
+{
+    [DotvvmControlCapability()]
+    public sealed class RouteLinkCapability
+    {
```

Comment:
I think we could add this capability to the built-in RouteLink control.

Reply:
Maybe put this in a separate PR.